Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -1195,10 +1195,25 @@ public async Task<ReconciliationResult> ReconcileAsync(OperationContext context)
 
                     while (!isFinished)
                     {
-                        var delayTask = Task.Delay(_configuration.ReconciliationCycleFrequency);
+                        var delayTask = Task.Delay(_configuration.ReconciliationCycleFrequency, context.Token);

```

Comment:
One minor thing: does it make sense to remove 'delayTask' local altogether and just have:

```
if (!isFinished) await Task.Delay(...)
```

Reply:
I just wanted to make sure that we really did cycles every 30 minutes, instead of every (30 minutes + cycle duration)